RealmTasks Apple/RealmTasks iOS/TablePresenter.swift

```diff
@@ -115,9 +115,9 @@ UITableViewDataSource, UITableViewDelegate, UIGestureRecognizerDelegate where Pa
     // MARK: UITableViewDelegate
 
     func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
-        if cellPresenter.currentlyEditingIndexPath?.row == indexPath.row {
-            return floor(cellPresenter.cellHeight(forText: cellPresenter.currentlyEditingCell!.textView.text))
-        }
+//        if cellPresenter.currentlyEditingIndexPath?.row == indexPath.row {
```

Comment:
Yeah, I'm not sure why that was in there, but it was interfering with my logic of growing the height of the table cell when the date label is visible.

I'm not sure what purpose it originally served, but in my basic testing, commenting it out didn't seem to break anything. But since I'm still not sure what purpose it was serving, I didn't completely delete it.